proto/beacon/p2p/v1/types.proto

```diff
@@ -16,9 +18,7 @@ message BeaconState {
   bytes randao_mix_hash32 = 1001;
   bytes next_seed_hash32 = 1002;
   repeated ShardAndCommitteeArray shard_and_committees_for_slots = 1003;
-  // TODO: Persistent committees should be a 2d uint24 array.
-  repeated uint32 persistent_committees = 1004 [deprecated=true];
-  // repeated Uint32Array persistent_committees = 1004;
+  repeated ethereum.common.Uint32List persistent_committees = 1004;
```

Comment:
Any thoughts on List vs Array? We use array above @nisdas @terenc3t @rauljordan

Reply:
I think a list is fine here